project4/index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Abril+Fatface|Cinzel|Playfair+Display" rel="stylesheet">
+    <title>SustainableConference</title>
```

Comment:
This element allows for spaces, so maybe consider something closer to your conference name?
```suggestion
    <title>Creating a World of Sustainable Graphic Design</title>
```